components/datadog/agent/kubernetes_helm.go

```diff
@@ -294,6 +294,18 @@ func buildLinuxHelmValues(baseName, agentImagePath, agentImageTag, clusterAgentI
 					"name":  pulumi.String("DD_TELEMETRY_CHECKS"),
 					"value": pulumi.String("*"),
 				},
+				pulumi.StringMap{
+					"name":  pulumi.String("placeholder_env"),
+					"value": pulumi.String("placeholder_val"),
+				},
+				pulumi.StringMap{
+					"name":  pulumi.String("DD_KUBERNETES_RESOURCES_LABELS_AS_TAGS"),
+					"value": pulumi.JSONMarshal(getResourcesLabelsAsTags().toJSONString()),
+				},
+				pulumi.StringMap{
+					"name":  pulumi.String("DD_KUBERNETES_RESOURCES_ANNOTATIONS_AS_TAGS"),
+					"value": pulumi.JSONMarshal(getResourcesAnnotationsAsTags().toJSONString()),
+				},
```

Comment:
Ideally, it should be passed as a helm variable. 

However, this is not possible currently because the feature hasn't yet been publicly added to the helm chart, so we can't use it here. 

The implementation can be changed here once we release the feature publicly in helm